versions.yaml

```diff
@@ -15,45 +15,88 @@
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
 _variables:
-  from: &from gcr.io/google-appengine/debian8
+  from9: &from9 gcr.io/google-appengine/debian9:latest
+  from8: &from8 gcr.io/google-appengine/debian8:latest
   gpg: &gpg A4A9406876FCBD3C456770C88C718D3B5072E1F5
   gosu: &gosu
     version: '1.10'
     gpg: B42F6819007F00F88E364FD4036A9C25BF357DD4
 
 versions:
-  - dir: '8/8.0'
+  - dir: '8/debian9/8.0'
     repo: mysql8
     tags:
+      - '8.0.3-debian9'
+      - '8.0-debian9'
       - '8.0.3'
       - '8.0'
       - 'latest'
-    from: *from
+    from: *from9
     packages:
       gosu: *gosu
       mysql:
-        version: 8.0.3-rc-1debian8
+        version: 8.0.11-1debian9
```

Comment:
Version mismatch, 8.0.3 in tag vs 8.0.11 in package

Reply:
Also, there is no 8.0.11 in Debian 8.

Reply:
Tags in version for Debian 9 - done
I do not touch version for Debian 8, this commit is only adding support for Debian 9

Reply:
The old directories have been deleted.